### ISSUE TITLE
SVG height and width

### ISSUE THREAD

**Issue by LincMitch** [airbnb/react-sketchapp#269]
I'm using react-primitives-svg to render the same SVG code to both web and sketch:

```
switch (Platform.OS) {
  case "sketch":
    SvgLib = require("react-primitives-svg/lib/core.sketch");
    break;
  case "web":
    SvgLib = require("react-primitives-svg/lib/core.web");
    break;
  default: // React Native should be able to pick up `core.ios.js` or `core.android.js` according to the platform
    // SvgLib = require("react-primitives-svg");
}
```

Now the SVG height and width seem to be interpreted differently for Sketch and web.
`<Svg viewBox="0 0 200 200" width="200" height="200">` is more like 20/20 on web, but OK in Sketch.
but...
`<Svg viewBox="0 0 200 200">` renders 95.61px /.47px in Sketch, but OK on web.

So I can't use the same code.

@alampros suggests it maybe a yoga layout issue as he had a similar problem #204

**Comment by VinSpee** [comment]
I think I'm banging into this same issue:

Code:

```jsx
const Bg = ({ type }: { type: string }) => (
  <Svg
    id="grad"
    style={{
      position: 'absolute',
      ...StyleSheet.absoluteFillObject,
      width: '100%',
      height: '100%',
    }}
  >
    <Defs>
      <LinearGradient
        id={`button-sheen-${type}`}
        x1="0"
        x2="0"
        y1="0"
        y2="1"
      >
        <Stop
          offset="0%"
          stopColor={THEME[type.toUpperCase()].BG_COLOR_START}
        />
        <Stop
          offset="100%"
          stopColor={THEME[type.toUpperCase()].BG_COLOR_END}
        />
      </LinearGradient>
    </Defs>
    <Rect
      fill={`url(#button-sheen-${type})`}
      style={{
        position: 'absolute',
        ...StyleSheet.absoluteFillObject,
        width: '100%',
        height: '100%',
      }}
      x="0"
      y="0"
    />
  </Svg>
);
```



Web screencap (yay!):
<img width="118" alt="screen shot 2018-04-27 at 2 18 27 pm" src="https://user-images.githubusercontent.com/582828/39378382-3c4f46b0-4a26-11e8-9a88-e663839e0af0.png">

Sketch screencap (no gradient?):
<img width="101" alt="screen shot 2018-04-27 at 2 18 11 pm" src="https://user-images.githubusercontent.com/582828/39378396-46bf17ba-4a26-11e8-8506-077e037ead5a.png">


Sketch gradient detail (ahh, there's a gradient it's just TINY?):
<img width="456" alt="screen shot 2018-04-27 at 2 19 12 pm" src="https://user-images.githubusercontent.com/582828/39378378-387ec3d0-4a26-11e8-8695-a8026963ac9f.png">

**Comment by isiaapp** [comment]
@VinSpee have you solved this problem?

**Comment by VinSpee** [comment]
@isiaap no I have not 😢

**Comment by isiaapp** [comment]
@VinSpee does it mean that you have refused to use gradients? Or some workaround was found?

**Comment by VinSpee** [comment]
@isiaapp yes, not using gradients for now. certainly not ideal or acceptable, but such is life.

**Comment by VinSpee** [comment]
Hi! It's been quite some time and a couple of large version bumps - is there any intent to fix this issue? I'm just curious if it's even on the roadmap. "No" is a completely acceptable and sane answer.
